emuu-client/src/components/Sidebar/Sidebar.js

```diff
@@ -218,16 +256,22 @@ export default function MiniDrawer({ sideBarState }) {
                         justifyContent: "center",
                       }}
                     >
-                      <Avatar />
+                      <Avatar
+                        src={
+                          user.ProfilePictureUrl
+                            ? user.ProfilePictureUrl
+                            : "https://wallpaperaccess.com/full/170249.jpg"
```

Comment:
Is this a stock image? If so, we already have a default one set when an account is made.